packages/dialtone-vue2/components/datepicker/modules/calendar.vue

```diff
@@ -30,7 +30,7 @@
           class="d-datepicker__cell"
         >
           <dt-button
-            :ref="el => { if (el) setDayRef(el, day) }"
```

Comment:
hmm yeah not sure this function within ref syntax is supported in vue 2.. still strange it works fine in storybook and not product though. Maybe it's a vue 2.7 vs vue 2.6 thing.

Reply:
Huh... I didnt think that product has a different vue 2 version that the one we are delivering solutions and components...

Reply:
Yep they are still on 2.6. Maybe we should revert dialtone back to 2.6 because this could definitely cause problems.